grimoire_elk/enriched/gitlab.py

```diff
@@ -319,7 +321,7 @@ def __get_rich_merge(self, item):
         rich_mr['created_at'] = merge_request['created_at']
         rich_mr['updated_at'] = merge_request['updated_at']
         rich_mr['url'] = merge_request['web_url']
-        rich_mr['merged'] = True if rich_mr['state'] else False
+        rich_mr['merged'] = True if rich_mr['state'] == 'merged' else False
```

Comment:
This should be done this way:

```suggestion
        rich_mr['merged'] = rich_mr['state'] == 'merged'
```

Reply:
I believe `True if rich_mr['state'] == 'merged' else False` was clearer, however now the code has been changed according to your comment

Reply:
This case is an anti-pattern (https://ucsb-cs24.github.io/topics/antipatterns/)